src/auth/auth.py

```
@@ -186,7 +186,7 @@ def wrapper(*args, **kwargs):
             token = get_token_auth_header()
             try:
                 payload = verify_decode_jwt(token)
-            except BaseException:
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)